security/pkg/server/ca/server.go

```diff
@@ -141,8 +141,8 @@ func New(ca ca.CertificateAuthority, ttl time.Duration, forCA bool, hostlist []s
 	authenticators := []authenticator{&clientCertAuthenticator{}}
 	// Temporarily disable ID token authenticator by resetting the hostlist.
 	// [TODO](myidpt): enable ID token authenticator when the CSR API authz can work correctly.
-	hostlist = make([]string, 0)
-	for _, host := range hostlist {
+	hostlistForJwtAuth := make([]string, 0)
+	for _, host := range hostlistForJwtAuth {
```

Comment:
Why we need a for loop for this empty array?

Reply:
hostlistForJwtAuth is temporarily reset to empty array. It will be fixed when ID token authenticator is enabled :)

Reply:
But we define this `hostlistForJwtAuth` new variable here. Where we get it from before disabling it?

Reply:
hostlistForJwtAuth is equal to hostlist before disabling it.

Reply:
Yes, we could. But unfortunately Golang will complain all the ID token auth related functions as unused. Then I have to comment out a lot of code and tests which doesn't seem very clean either :P I can do that if you think that's the right way, though.